flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/stream/table/AggregateITCase.scala

```diff
@@ -203,13 +203,13 @@ class AggregateITCase extends StreamingWithStateTestBase {
       .groupBy('b)
       .select('a.count as 'cnt, 'b)
       .groupBy('cnt)
-      .select('cnt, 'b.count as 'freq)
+      .select('cnt, 'b.count as 'freq, 'b.min as 'min, 'b.max as 'max)
 
     val results = t.toRetractStream[Row](queryConfig)
 
     results.addSink(new RetractingSink)
     env.execute()
-    val expected = List("1,1", "2,1", "3,1", "4,1", "5,1", "6,1")
+    val expected = List("1,1,1,1", "2,1,2,2", "3,1,3,3", "4,1,4,4", "5,1,5,5", "6,1,6,6")
```

Comment:
I had no closer look at the PR yet. But what I see is little test coverage. Are there existing tests that properly test the changes?

Reply:
@twalthr Thanks a lot for the review. I think the modified testDoubleGroupAggregation can cover the changes.

Reply:
this test does not cover statebackend usage, a prove for this is that this test passes before your last commit.

Reply:
Yes we can also do this in another PR. In general we definitely need more harness tests. For example, FLINK-10674 could have never been found without proper unit tests. It was luck that the distinct accumulator threw a NullPointerException.

Reply:
Agree. I have created another ticket https://issues.apache.org/jira/browse/FLINK-11074 and will add detailed harness tests for all the built-in aggregate functions which use DataView.

Reply:
Yeah. I am incline to suggest we fix FLINK-11074 first before we check this in. What do you think @dianfu @twalthr .

Reply:
@walterddr @twalthr Thanks for the suggestion and I have created a PR for FLINK-11074: https://github.com/apache/flink/pull/7253.